packages/puppeteer-core/src/cdp/WebWorker.ts

```diff
@@ -117,6 +118,21 @@ export class WebWorker extends EventEmitter<Record<EventType, unknown>> {
     this.#client.send('Runtime.enable').catch(debugError);
   }
 
+  /**
+   * @internal
+   */
+  set terminated(value: boolean) {
+    this.#terminated = value;
+  }
+
+  /**
+   * Whether the {@link WebWorker}
+   * {@link https://developer.mozilla.org/en-US/docs/Web/API/Worker/terminate | terminated}.
+   */
+  get terminated(): boolean {
```

Comment:
missing test coverage for terminated and it does not appear to be used internally?